harness/determined/tensorpack/_tensorpack_trial.py

```diff
@@ -453,6 +463,7 @@ def _init_model(self, training_dataflow: Any, validation_dataflow: Any) -> None:
             self.workloads,
             self.is_chief,
             self.rendezvous_info.get_rank(),
+            self,
```

Comment:
non-blocking: can we pass just the context here, when we end pass the whole trial controller in, the callback eventually being way too coupled with the controller (i.e. EstimatorTrial).

Reply:
Yeah I was debating about this but followed Estimator trials lead, will make this change!